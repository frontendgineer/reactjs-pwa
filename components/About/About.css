.heading {
    font-size: 3rem;
}
.bio {
    font-size: 1.8rem;
    line-height: 1.7;
    color: #333;
}

.container {
    display: flex;
    justify-content: space-between;
    max-width: var(--site-width);
    margin: 0 auto;
}
