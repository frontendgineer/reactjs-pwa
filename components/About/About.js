import React from 'react'
import styles from './About.css'
import {Link} from 'react-router-dom'

const About = () => (
    <div>
        <h1><Link to='/'>HTC Corporation</Link></h1>
        <p className={styles.bio}>Doloremque donec. Eros posuere adipiscing, incidunt, lectus quis, incidunt esse sem, suspendisse, quibusdam consequatur! Occaecat! Sollicitudin? Adipisicing quae lorem commodo aperiam. Officia quaerat minim explicabo reprehenderit incididunt aptent, aut blandit montes. Tristique magna quaerat ante irure euismod natoque viverra autem, faucibus, cupiditate wisi rem interdum aliquet soluta diamlorem elit convallis quibusdam malesuada montes, maiores voluptate asperiores? Ipsa quam laborum, aptent! Dui exercitation nobis? Ea! Sociosqu? Labore dictum dolor distinctio arcu, atque dignissimos harum nulla! Pariatur pulvinar curabitur totam nemo aute ad officia eu consectetur, voluptatem, blandit earum perspiciatis aspernatur rerum, primis potenti. Sapien class cubilia, etiam, eiusmod debitis, explicabo voluptates.</p>
    </div>
)

export default About
