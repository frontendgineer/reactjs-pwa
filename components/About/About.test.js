import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'
import About from './About'

test('it should render About render', () => {
    const component = shallow(<About />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
})
