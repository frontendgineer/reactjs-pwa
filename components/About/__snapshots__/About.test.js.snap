// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render About render 1`] = `
<div>
  <h1>
    <Link
      replace={false}
      to="/"
    >
      HTC Corporation
    </Link>
  </h1>
  <p
    className="bio"
  >
    Doloremque donec. Eros posuere adipiscing, incidunt, lectus quis, incidunt esse sem, suspendisse, quibusdam consequatur! Occaecat! Sollicitudin? Adipisicing quae lorem commodo aperiam. Officia quaerat minim explicabo reprehenderit incididunt aptent, aut blandit montes. Tristique magna quaerat ante irure euismod natoque viverra autem, faucibus, cupiditate wisi rem interdum aliquet soluta diamlorem elit convallis quibusdam malesuada montes, maiores voluptate asperiores? Ipsa quam laborum, aptent! Dui exercitation nobis? Ea! Sociosqu? Labore dictum dolor distinctio arcu, atque dignissimos harum nulla! Pariatur pulvinar curabitur totam nemo aute ad officia eu consectetur, voluptatem, blandit earum perspiciatis aspernatur rerum, primis potenti. Sapien class cubilia, etiam, eiusmod debitis, explicabo voluptates.
  </p>
</div>
`;
